content/applications/finance/fiscal_localizations/vietnam.rst

```diff
@@ -0,0 +1,58 @@
+=======
+Vietnam
+=======
+
+Add Vietnamese QR banking codes to invoices
+===========================================
+
+**Vietnamese QR banking** is a payment service platform that allows customers to make instant
+domestic payments to individuals and merchants in Vietnamese dong (VND) via online and mobile
+banking.
+
+Activate QR codes
+-----------------
+
+Go to :menuselection:`Accounting --> Configuration --> Settings`, and under the :guilabel:`Customer
+Payments` section, activate the :guilabel:`QR Codes` feature.
+
+QR codes bank account configuration
+-----------------------------------
+
+Go to :menuselection:`Contacts --> Configuration --> Bank Accounts` and select the bank account for
+which you want to activate **Vietnamese QR banking**. Set the :guilabel:`Bank Identifier Code` on
+the bank, then set the :guilabel:`Proxy Type` and fill in the :guilabel:`Proxy Value` field
```

Comment:
```suggestion
the bank, then set the :guilabel:`Proxy Type` and fill in the :guilabel:`Proxy Value` field
```
![proxy](https://github.com/odoo/documentation/assets/104567387/410ca63e-0807-49b1-b0da-246bdaed4569)

I couldn't find the "Proxy Type" field on the contact form. Is it still relevant ? :)

Reply:
Hello @toaa-odoo, May I ask which version are you using? This seems like an older version of the development

Reply:
The proxy type is still required.

Reply:
@toaa-odoo You could try this runbot: https://47551880-master-all.runbot104.odoo.com/

Reply:
I had used the one linked in the chatter of the task. The one you just provided work though! All good from me, I'll update @dade-odoo 's suggestion